sdk/keyvault/azure-security-keyvault-common/vcpkg/vcpkg.json

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+#
```

Comment:
I am wondering if we should remove these comment now, since the file is JSON, and JSON doesn't support comments.

Reply:
Is there any json-parser which will fail because if this file header?  
Are we auto-removing this with release-scripts anyway?

Reply:
We remove them with scripts

Reply:
> Is there any json-parser which will fail because if this file header?

Yes, if they ingest it directly. Many RFC-compliant json parsers trip up on this.

> Are we auto-removing this with release-scripts anyway?

Yes.